src/DDTrace/Integrations/Symfony/V4/SymfonyBundle.php

```diff
@@ -48,13 +52,18 @@ public function boot()
 
         // Create a span that starts from when Symfony first boots
         $scope = $tracer->startActiveSpan('symfony.request');
-        $symfony_request_span = $scope->getSpan();
-        $symfony_request_span->setTag(Tags\SERVICE_NAME, $this->getAppName());
-        $symfony_request_span->setTag(Tags\SPAN_TYPE, Types\WEB_SERVLET);
+        $symfonyRequestSpan = $scope->getSpan();
+        $symfonyRequestSpan->setTag(Tags\SERVICE_NAME, $this->getAppName());
+        $symfonyRequestSpan->setTag(Tags\SPAN_TYPE, Types\WEB_SERVLET);
 
         // public function handle(Request $request, $type = HttpKernelInterface::MASTER_REQUEST, $catch = true)
-        dd_trace(HttpKernel::class, 'handle', function ($request, ...$args) use ($symfony_request_span) {
+        dd_trace(HttpKernel::class, 'handle', function ($request, ...$args) use ($symfonyRequestSpan) {
             $scope = GlobalTracer::get()->startActiveSpan('symfony.kernel.handle');
+            $symfonyRequestSpan->setTag('request.method', $request->getMethod());
```

Comment:
I'm not sure if you guys have plans regarding meta tag names request attributes. Maybe you already have on other projects. If there are any guidelines on this, please let me know.

Reply:
Hey! Thanks for the contribution, really appreciated!

> I'm not sure if you guys have plans regarding meta tag names request attributes

Do you mean how to to include request's query string parameters?

Reply:
Query string, request method, uri, content-type etc... 
I guess not all will make sense to everyone, some people might don't want to see content-type but are more interested in some other custom headers to be added as meta to the trace.

Reply:
> request method and uri

You were right to add.

> Query string

This could be a little bit more tricky, as it may leak some data. Hopefully nobody add "auth token" tot he query string, but I have seen them. So please hold on on this for now, we may want to provide some sort of configuration/hook to do that.

> content-type etc

For headers in general we have an RFC which is in final review stage to provide a consistent way across all the languages and integrations that we support to report http headers (and thus content-type among them). So for now I would prefer not to add the content type. While I cannot commit to a date, is will be something that we implement not far from now.

Reply:
I removed the query string and content type.
HTTP meta now looks like this:
```
http { 
  method: GET
  url: http://localhost.lh:9091/ping
} 
```